samples_project/Assets/SampleViewer/Samples/StreamLayer/StreamLayerWebSocketSubscribe.cs

```diff
@@ -88,7 +88,7 @@ public class StreamLayerWebSocketSubscribe : MonoBehaviour
     // The height where we spawn the flight before finding the actual height
     private int FlightSpawnHeight = 10000;
 
-    public string wsUrl = "ws://geoeventsample1.esri.com:6180/arcgis/ws/services/FAAStream/StreamServer/subscribe";
+    public string wsUrl = "wss://geoeventsample1.esri.com:6143/arcgis/ws/services/FAAStream/StreamServer/subscribe";
```

Comment:
The ws URL still valid and works as expected. But the WebSocket URL on the page below is a wss URL so it might be nice for users to be clear.

https://geoeventsample1.esri.com:6443/arcgis/rest/services/FAAStream/StreamServer